Sources/App/Extensions/Vapor.Request+RequestProtocol.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+import Bot
+import Vapor
+
+extension Vapor.Request: RequestProtocol {
+    public func header(named name: String) -> String? {
+        return http.headers["X-GitHub-Event"].first
```

Comment:
The `name` parameter isn't used there, and instead `X-GitHub-Event` is hardcoded, seems like an oversight?

```suggestion
        return http.headers[name].first
```

Reply:
Great catch Olivier 👍